.travis.yml

```diff
@@ -5,6 +5,12 @@ sudo: false
 services:
   - postgresql
 
+addons:
+  apt:
+    packages:
+      - postgresql-9.1
```

Comment:
oops sorry, this should be combined `postgresql-9.1-postgis-2.1`

Reply:
That is a big name :smile: 
Fixed!